### PR TITLE
Update the config doc

### PR DESCRIPTION
**Pull request by chicoxyzzy**
related socket-sdk PR https://github.com/socketsupply/socket-sdk/pull/372#pullrequestreview-1093292660

### CODE REVIEW

File: src/docs/config.md [hunk 1]
```diff
@@ -4,31 +4,29 @@
...
 | `height` | | | The initial height of the first window |
 | `input` | | | A directory is where your application's code is located |
-| `lang` | | | Localization |
-| `maintainer` | | | |
+| `lang` | | en-us | Localization |
+| `maintainer` | | | A String used in the about dialog and meta info. |
 | `name` | ![check](/images/icons/checkmark.svg) | | The name of the program |
-| `output` | ![check](/images/icons/checkmark.svg) | | The binary output path |
+| `output` | ![check](/images/icons/checkmark.svg) | dist | The binary output path. It's recommended to add this path to .gitignore. |
+| `revision` | | 1 | TODO: maybe the user doesn’t need to know about this? |
```
Comment (chicoxyzzy) [inline src/docs/config.md:19]: should have a better description

File: src/docs/config.md [hunk 2]
```diff
@@ -40,40 +38,42 @@
...
 
+[macOS code signing guide][/guides/#macos-1]
+
 | Property | Required | Default | Description |
 | :--- | :--- | :--- | :--- |
 | `mac_appstore_icon` | | | Mac App Store icon |
 | `mac_category` | | | A category in the App Store |
-| `mac_cmd` | ![check](/images/icons/checkmark.svg) | | The command to execute to spawn the “back-end” process. |
-| `mac_distribution_method` | | | |
-| `mac_entitlements` | | | plist file path |
```
Comment (chicoxyzzy) [inline src/docs/config.md:60]: not used

File: src/docs/config.md [hunk 3]
```diff
@@ -40,40 +38,42 @@
...
-| `mac_icon` | ![check](/images/icons/checkmark.svg) | | The icon to use for identifying your app on MacOS. |
-| `mac_provisioning_profile` | | | |
```
Comment (chicoxyzzy) [inline src/docs/config.md:62]: not used

File: src/docs/config.md [hunk 4]
```diff
@@ -40,40 +38,42 @@
 | Property | Required | Default | Description |
 | :--- | :--- | :--- | :--- |
 | `linux_categories` | | | Helps to make your app searchable in Linux desktop environments |
-| `linux_cmd` | ![check](/images/icons/checkmark.svg) | | The command to execute to spawn the “back-end” process. |
-| `linux_icon` | ![check](/images/icons/checkmark.svg) | | The icon to use for identifying your app in Linux desktop environments. |
+| `linux_cmd` | | | The command to execute to spawn the “back-end” process. |
+| `linux_icon` | | | The icon to use for identifying your app in Linux desktop environments. |
+| `linux_log_stdout` | | false | A boolean that determines if WebKit WebView logs should get forwarded to stdout and stderr |
```
Comment (jwerle) [inline src/docs/config.md:43]: maybe `linux_forward_console_to_stdout` here too?

**Review by jwerle — APPROVED** [review]

**Pull request merged**